contrib/debug-tools/docs/remote-debug-pai-job.md

```diff
@@ -0,0 +1,85 @@
+# 1. Overview
+
+OpenPai remote debug tool only support **python** job. It contains two debug modes:
+
+1. Start remote debug when job task starts
+2. Inject breakpoint in source code and start debug when hit the breakpoint.
+
+Currently this feature is in experiment phrase, and we don't have a mechanism to notify user weather debug server is starts. Users need to check log to my theirself to get this notification.
```

Comment:
Users need to check log to my theirself to get this notification -> Users need to check log themselves get this notification

Reply:
fixed